src/components/locations/constants.js

```diff
@@ -27,11 +27,11 @@ export const NEW_LOCATION = 'New Location';
 /** @constant */
 export const CREATE_LOCATION = 'Create';
 /** @constant */
-export const UPLOAD_TEXT = 'Upload Images';
+export const UPLOAD_TEXT = 'Images';
 /** @constant */
 export const CREATE_NEW_LOCATION = 'Create New Location';
 /** @constant */
-const baseUrl = 'https://europe-west1-neonatar.cloudfunctions.net/api';
+const baseUrl = 'http://localhost:5000/neonatar/europe-west1/api';
 export const LOCATION_URL = `${baseUrl}/v1/locations`;
```

Comment:
I suppose you used localhost while you were still testing locally. Please change to deployed functions if everything works fine.

Reply:
Alright, will do that